### ISSUE TITLE
[2.0] Multiple entries in summary

### ISSUE THREAD

**Issue by ille** [anshooarora/extentreports#9]
Using:
Extenreports 2.0
Junit 4.12.

start tests in the @Before with 
```java
test = extent.startTest(name.getMethodName());
```
end tests in @After with
```java
extent.endTest(test);
extent.flush();
```
Getting multiple entries of same tests in the summary. (looks like it writes them on every flush.)

summary example:
test01
test01
test02
test01
test02
test03
test01
test02
test03
test04

**Comment by anshooarora** [comment]
Please try version 2.01, it has just been uploaded to http://relevantcodes.com/extentreports-for-selenium/.  This has been fixed.

**Comment by ille** [comment]
fixed in 2.01
